src/api/pursuances.js

```diff
@@ -3,7 +3,7 @@ import { PURSUANCE_DISPLAY_PREFIX } from '../constants';
 
 export const getPursuancesReq = () => {
   return postgrest
-    .getJSON('/pursuances')
+    .getJSON('/pursuances?select=id,name,mission')
```

Comment:
@moeshoman Let's just use `/pursuance`, since it's not that much data anyway, and if we make schema changes then we won't have to change this line/query :+1: